packages/SwingSet/test/test-vpid-liveslots.js

```diff
@@ -473,7 +473,7 @@ async function doVatResolveCase23(t, which, mode, stalls) {
 
   // assert that the vat saw the local promise being resolved too
   if (mode === 'presence') {
-    t.is(resolutionOfP1.toString(), `[Alleged: presence ${target2}]`);
+    t.is(`${q(resolutionOfP1)}`, `"[Alleged: presence ${target2}]"`);
```

Comment:
Does normal `{resolutionOfP1}` get stringified the same way? I wouldn't want to require developers to use `q()` during debugging. Ordinary `toString` must keep working, even if you can't rely upon the value for some purposes.

Reply:
Does `resolutionOfP1` inherit from `Object.prototype`? If so, it'll get stringified to the expansion of
```js
`[object presence ${target2}]`
```
which is similar. However, if that worked, the original line that I'm replacing would have worked, which is why I'm suspicious. Where is this object created? How is it created?

Reply:
I figured out why stringifying those test case objects with `o.toString()` was failing. `${o}` would have failed the same way. They were indeed not inheriting from `Object.prototype`, because that's how presences are currently made. IThis PR also changes `Remotable` and `Far` to change inheriting from `null` to inheriting from `Object.prototype`. Far object now always inherit from something.

I also reverted my changes to the kernel tests.

Reply:
I still need to change the kernel tests to test against the new stringified rather than the old one.